x-pack/plugins/security_solution/public/common/components/navigation/security_side_nav/security_side_nav.tsx

```diff
@@ -0,0 +1,155 @@
+
+  const formatSideNavItem: FormatSideNavItems = useCallback(
+    (navLinkItem) => {
+      const formatDefaultItem = (navItem: NavLinkItem): DefaultSideNavItem => ({
+        id: navItem.id,
+        label: navItem.title,
+        ...getSecuritySolutionLinkProps({ deepLinkId: navItem.id }),
+        ...(navItem.categories && navItem.categories.length > 0
+          ? { categories: navItem.categories }
+          : {}),
+        ...(navItem.links && navItem.links.length > 0
+          ? {
+              items: navItem.links
+                .filter((link) => !link.disabled)
+                .map((panelNavItem) => ({
+                  id: panelNavItem.id,
+                  label: panelNavItem.title,
+                  description: panelNavItem.description,
+                  ...getSecuritySolutionLinkProps({ deepLinkId: panelNavItem.id }),
+                })),
+            }
+          : {}),
+      });
+
+      const formatGetStartedItem = (navItem: NavLinkItem): CustomSideNavItem => ({
+        id: navItem.id,
+        render: (isSelected) => (
+          <GetStartedCustomLink isSelected={isSelected} title={navItem.title} />
+        ),
+      });
+
+      if (navLinkItem.id === SecurityPageName.landing) {
+        return formatGetStartedItem(navLinkItem);
+      }
+      return formatDefaultItem(navLinkItem);
+    },
+    [getSecuritySolutionLinkProps]
+  );
+
+  return formatSideNavItem;
+};
+
+/**
+ * Returns the formatted `items` and `footerItems` to be rendered in the navigation
+ */
+const useSideNavItems = () => {
+  const appNavLinks = useAppNavLinks();
+  const formatSideNavItem = useFormatSideNavItem();
+
+  const sideNavItems = useMemo(() => {
+    const mainNavItems: SideNavItem[] = [];
+    const footerNavItems: SideNavItem[] = [];
+    appNavLinks.forEach((appNavLink) => {
+      if (appNavLink.disabled) {
+        return;
+      }
+      if (isFooterNavItem(appNavLink.id)) {
+        footerNavItems.push(formatSideNavItem(appNavLink));
+      } else {
+        mainNavItems.push(formatSideNavItem(appNavLink));
+      }
+    });
+    return [mainNavItems, footerNavItems];
+  }, [appNavLinks, formatSideNavItem]);
+
+  return sideNavItems;
+};
+
+const useSelectedId = (): SecurityPageName => {
+  const [{ pageName }] = useRouteSpy();
+  const selectedId = useMemo(() => {
+    const [rootLinkInfo] = getAncestorLinksInfo(pageName as SecurityPageName);
+    return rootLinkInfo?.id ?? '';
+  }, [pageName]);
+
+  return selectedId;
+};
+
+/**
+ * Main security navigation component.
+ * It takes the links to render from the generic application `links` configs.
+ */
+export const SecuritySideNav: React.FC = () => {
+  const [items, footerItems] = useSideNavItems();
+  const selectedId = useSelectedId();
+
+  // TODO: move this exclusion logic to getManagementLinkItems
+  const canSeeHostIsolationExceptions = useCanSeeHostIsolationExceptionsMenu();
```

Comment:
Hey @dasansol92, As suggested by @semd we are rolling back the change that removes `HostIsolationExceptions` from AppLinks. Here is the commit 28b90678ff5efea7f9beda2bbab36a4fc450b316

Reply:
Awesome, so it will be hidden by default and displayed if needed right?

Reply:
It will render on the security solution menu and manage the landing page when `useCanSeeHostIsolationExceptionsMenu` return true and hidden when it returns false. It should work similarly to the old menu: https://github.com/elastic/kibana/blob/a0cf4506d1806af7a9380ff73f681bc611126d4b/x-pack/plugins/security_solution/public/common/components/navigation/use_security_solution_navigation/use_navigation_items.tsx#L119